Dockerfile

```diff
@@ -56,6 +57,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 		patchelf \
 		php \
 		pkg-config \
+		python2 \
```

Comment:
Are python3-venv and python3-pip not necessary here? I think python3-apt is also necessary for some packages to install their own depends. I'm not 100% familiar with the rest of the process in this docker.

Reply:
No they're not needed.  How `spksrc` works is that it will build its own native python exactly matching the cross-compiled one that is built right afterwards.  So the host python always matches the target python.  Also, the other necessary tools such as `pip`, `crossenv`, etc. all get's intalled within either the native or cross versions of python, depending where it's needed.